demo/ContainerViewController.swift

```diff
@@ -390,11 +393,11 @@ private extension SafetyState.ObjectType {
 
 extension ContainerViewController: NavigationManagerDelegate {
     
-    func navigationManager(_ navigationManager: NavigationManager, didUpdate route: MapboxVisionAR.Route) {
+    func navigationService(didUpdate route: MapboxVisionAR.Route) {
         delegate?.didNavigationRouteUpdated(route: route)
     }
     
-    func navigationManagerArrivedAtDestination(_ navigationManager: NavigationManager) {
+    func navigationServiceArrivedAtDestination() {
```

Comment:
Please rename func and use past simple tense: `…DidArriveAtDestination`.

Reply:
done